docs/src/main/sphinx/release/release-446.md

```diff
@@ -0,0 +1,67 @@
+# Release 446 (29 Apr 2024)
+
+## General
+
+* Improve performance of `INSERT` statements into partitioned tables when the
+  `retry_policy` configuration property is set to `TASK`. ({issue}`21661 `)
+* Reduce delay before stopping queries when the cluster runs out of memory. ({issue}`21719`)
+* Fix catalog names including `"` marks when creating them with `CREATE CATALOG`
+  statements. ({issue}`21399`)
+* Fix query failure when a token in a query ends with a `:`. ({issue}`21676`)
+* Fix potential query failure for queries using [SQL routines](/routines) with
+  the `LEAVE`, `ITERATE`, `REPEAT`, or `WHILE` labels. ({issue}`21682`)
+* Fix potential query failure for queries using [SQL routines](/routines) that
+  invoke other functions. ({issue}`19820`)
+* Fix potential query failure due to worker nodes running out of memory in
+  concurrent scenarios. ({issue}`21706`)
+* Fix non-propagation of OpenTelemetry tracing context when invoking a
+  connector. ({issue}`21575`)
+
+## BigQuery connector
+
+* {{breaking}} Enable `bigquery.arrow-serialization.enabled` by default.
+  `--add-opens=java.base/java.nio=ALL-UNNAMED` must be added to`jvm-config`. ({issue}`21580`)
+
+## Delta Lake connector
+
+* Fix query failure when querying Azure file storage with special characters in
+  a schema, table, or column name. ({issue}`21586`)
+* Fix incorrect results with a deletion vector on a partitioned table. ({issue}`21737`)
+
+## Hive connector
+
+* Add support for reading S3 objects restored from Glacier storage. ({issue}`21164`)
+* Fix query failure when querying Azure file storage with special characters in
+  a schema, table, or column name. ({issue}`21586`)
+
+## Iceberg connector
+
+* Add support for the [Snowflake catalog](iceberg-snowflake-catalog). ({issue}`19362`)
+* Automatically use `varchar` as a type during table creation when `char` is
+  specified. ({issue}`19336`, ({issue}`21515`)
+* Deprecate the `schema` and `table` arguments for the `table_changes` function
+  in favor of `schema_name` and `table_name`, respectively. ({issue}`21698`)
+* Prevent assigning `null` values to non-null columns as part of a `MERGE`
```

Comment:
I think since you use non-null .. you should also just use null without apostrophes